third_party/vizdoom/vizdoom.BUILD

```diff
@@ -715,9 +721,11 @@ cc_binary(
         "-msse",
         "-msse2",
         "-mmmx",
+        "-include glibc.h"
```

Comment:
should be like "-include=$(execpath :glibc.h)"

Reply:
in copts attribute of cc_binary rule @vizdoom//:vizdoom: label '@vizdoom//:glibc.h' in $(location) expression is not a declared prerequisite of this rule